src/main/java/seedu/address/logic/parser/FindStarsLessThanCommandParser.java

```diff
@@ -0,0 +1,36 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.logic.commands.FindStarsLessThanCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.person.StarsLessThanPredicate;
+
+/**
+ * Parser for the FindStarsLessThanCommand.
+ */
+public class FindStarsLessThanCommandParser implements Parser<FindStarsLessThanCommand> {
+    /**
+     * Parses the given {@code String} of arguments in the context of the FindStarsLessThanCommand
+     * and returns an FindStarsLessThan object for execution.
+     *
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public FindStarsLessThanCommand parse(String args) throws ParseException {
+        requireNonNull(args);
+        try {
+            String[] splitArgs = args.split(" ");
+            int intValue = Integer.parseInt(splitArgs[1]);
```

Comment:
Functionally there's no issue, but I'm not sure whether this is the best way to parse out the number. Imo it would be more KISS to do `intValue = Integer.parseInt(args.trim());`

Pros:

1. No need to catch the `ArrayIndexOutOfBoundsException`
2. inputs like `findStarsLT        1` with multiple spaces will work unlike the above.
3. Intent is clearer (to me).

Although I think this wld be up to your intended behaviour for the case of (2.) so wouldn't say it's necessary to change this

Reply:
This makes sense to me as well, will update accordingly.